concert/experiments/addons.py

```diff
@@ -137,14 +137,14 @@ class ImageWriter(Addon):
 
     A :class:`~concert.storage.Walker` instance
 
-    .. py:attribute:: async
+    .. py:attribute:: casync
```

Comment:
It does not break it, but I think here happend a little bit to much find-and-replace.

Reply:
Yeah I will polish these things.